.travis.yml

```diff
@@ -37,7 +37,7 @@ matrix:
 before_install:
   - echo "HOME="
   - echo $HOME
-  - sudo apt-get update -qq; sudo apt-get install -qq libopenmpi-dev; echo
+  - sudo apt-get update -qq; echo
```

Comment:
I'm not sure what the `echo` at the end is used for. Is it necessary? If that echo is needed, it looks good to me.

Reply:
indeed, not needed! thx for the review